src/store/state/state-model.ts

```diff
@@ -1,5 +1,10 @@
 import { StateModelIF } from '@/interfaces'
 
 export const stateModel: StateModelIF = {
-  stateText: 'Base State Text'
+  stateText: 'Base State Text',
+  currentDate: null,
+  certifyState: {
+    certifyFormValid: false,
+    certifiedBy: null
+  }
```

Comment:
This state model was initially put in just as an example. With the setup we have going we can group specific and related properties into these models. That being said we can totally have a generic model for this type of data but would we want to rename it? Possible also removed the example data property `stateText` if we were to use this model for generic data.

Reply:
My understanding was this will hold the application state and any common properties like date etc which will be used by the whole application. Initially I thought of reorganizing the store into modules with namespaces. But that will be a bit too much because we are dealing with one application even though it has different steps. My preference would be using this to put anything that represents the current state of the application. We can rename it. I will remove the sample property

Reply:
Sounds good, totally agree!